resources/helm/v2.1/gateways/istio-ingress/templates/rolebindings.yaml

```diff
@@ -2,17 +2,17 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
-  name: {{ $gateway.name | default "istio-ingressgateway" }}-sds
-  namespace: {{ $gateway.namespace | default .Release.Namespace }}
+  name: {{ $gateway.name }}-sds
+  namespace: {{ .Release.Namespace }}
```

Comment:
This needs to be reverted, or it will break installation of gateways in other namespaces.